medical/tests/test_medical_patient.py

```diff
@@ -139,3 +142,21 @@ def test_check_birthdate_date(self):
                     now + timedelta(days=20),
                 )
             })
+
+    def test_search_age(self):
+        """
+        When patients are searched by age,
+        it should return patients with the corresponding birth dates
+        """
+        birthdate = datetime.strptime(
+            self.patient_1.birthdate, "%Y-%m-%d"
+        ).date()
+        current_date = date.today()
+        delta = current_date - birthdate
+        years = delta.days/365 - 1
+        result = self.env['medical.patient'].search(
+            [('age', 'like', years)]
```

Comment:
@lasley I made this change, although now my test is failing because of the concern I explained above. What is the best way to make the search use the operators you wanted: `('=', '>=', '>', '<', '<=')`? Should I make a new field that computes the age as an integer as then use that for the search?